rhel6/guide/system/software/integrity/rpm_verification/rpm_verify_permissions.rule

```diff
@@ -2,7 +2,7 @@ documentation_complete: true
 
 title: 'Verify and Correct File Permissions with RPM'
 
-description: "The RPM package management system can check file access\npermissions of installed software packages, including many that are\nimportant to system security. After locating a file with incorrect\npermissions which can be found with <pre>$ rpm -Va | grep '^.M'</pre>,\nrun the following command to determine which package owns it:\n<pre>$ rpm -qf <i>FILENAME</i></pre>\nNext, run the following command to reset its permissions to \nthe correct values:\n<pre>$ sudo rpm --setperms <i>PACKAGENAME</i></pre>"
+description: "The RPM package management system can check file access\npermissions of installed software packages, including many that are\nimportant to system security. After locating a file with incorrect\npermissions which can be found with <pre>$ rpm -Va | grep '^.M'</pre>,\nrun the following command to determine which package owns it:\n<pre>$ rpm -qf <i>FILENAME</i></pre>\nNext, run the following command to reset its permissions to \nthe correct values:\n<pre>$ sudo rpm --quiet --setperms <i>PACKAGENAME</i></pre>"
```

Comment:
I won't hold this PR because of this but why not reflow this so it's readable :-/

Reply:
Agree.